projects/wizard_of_wikipedia/mturk_evaluation_task/run.py

```diff
@@ -90,17 +92,22 @@ def inject_override(opt, override_dict):
             opt[k] = v
 
     def get_logger(opt):
-        logger = logging.getLogger()
-        logger.setLevel(logging.INFO)
-
-        fmt = logging.Formatter('%(asctime)s: [ %(message)s ]', '%m/%d/%Y %I:%M:%S %p')
-        console = logging.StreamHandler()
-        console.setFormatter(fmt)
-        logger.addHandler(console)
+        fmt = '%(asctime)s: [ %(message)s ]'
+        logfile = None
         if 'mturk_log' in opt:
-            logfile = logging.FileHandler(opt['mturk_log'], 'a')
-            logfile.setFormatter(fmt)
-            logger.addHandler(logfile)
+            logfile = opt['mturk_log']
+            if not os.path.isdir(os.path.dirname(logfile)):
+                raise OSError(
```

Comment:
rather than raise here, you can just run `os.makedirs(dir, exist_ok=True)` which will make the dir (`exist_ok` means that it will not error if the directory already exists)